vmc/resource_vmc_sddc.go

```diff
@@ -13,6 +13,8 @@ import (
 	"github.com/hashicorp/terraform/helper/schema"
 	"github.com/hashicorp/terraform/helper/validation"
 	"github.com/vmware/vsphere-automation-sdk-go/lib/vapi/std/errors"
+	nsxtapi "github.com/vmware/vsphere-automation-sdk-go/services/nsxt-vmc-aws-integration/api"
```

Comment:
probably better to rename this to nsxtintegrationapi or nsxtintapi just to distinguish from nsxt apis

Reply:
Sure. I'll update it

Reply:
Done